index.html

```diff
@@ -17,25 +17,30 @@
     <header class="nav-container">
         <img src="./images/mobile_menu.png" class="nav-icon" alt="Nav-bar Hamburger">
         <div class="log">
-            <ul>
-                <li><a href="https://facebook.com"> <img src="images/facebook.png" alt=""></a></li>
-                <li><a href="https://twitter.com"><img src="images/twitter.png" alt=""></a></li>
-                <li><a href="#">Logout</a></li>
-                <li><a href="#">Mypage</a></li>
-                <li><a href="#">English</a></li>
-            </ul>
-        </div>
-        <nav class="nav-bar">
-            <a href="index.html"><img src="images/wmflogo.jpg" alt="logo"></a>
-            <ul>
-                <li><a href="about.html">About</a></li>
-                <li><a href="#">Program</a></li>
-                <li><a href="#">Join</a></li>
-                <li><a href="#">Sponsor</a></li>
-                <li><a href="#">News</a></li>
-            </ul>
-            <span class="campaign"><a href="#">Campaign</a></span>
-        </nav>
+            <span></span>
+            <div class="log-link-container">     
+                <ul>
+                    <li><a href="https://facebook.com"> <img src="images/facebook.png" alt=""></a></li>
+                    <li><a href="https://twitter.com"><img src="images/twitter.png" alt=""></a></li>
+                    <li><a href="#">Logout</a></li>
+                    <li><a href="#">Mypage</a></li>
+                    <li><a href="#">English</a></li>
+                </ul>
+            </div>
```

Comment:
- [ Required ] As requested by the previous reviewer, I think it would be better if the menu items in the black top bar are aligned to the right and aligned in line with the `red` bordered button in the white main menu. Please see the screenshot below for more details on this and rectify it.

- [ Required ] As requested by the previous reviewer, I think it would be better if the `logo` is aligned in line with the `text` in the hero section. Please see the screenshot below for more details on this and rectify it.

## Your design
![Screenshot from 2023-08-03 22-55-05](https://github.com/BHObserver/module-one-capstone-project/assets/16392046/231c9ea6-a424-46e1-9ba0-5b389b0c2475)


## Project wireframe
![Screenshot from 2023-08-03 23-02-45](https://github.com/BHObserver/module-one-capstone-project/assets/16392046/93cbf3a3-d6ae-469e-a049-da928ffe89f2)

Reply:
@Felix45 Brother, it is aligned properly in my machine. What is screen size of your machine?